app/test_main.py

```diff
@@ -1,3 +1,17 @@
 from app.main import get_human_age
 
-# write your code here
+
```

Comment:
Consider using `@pytest.mark.parametrize` to avoid repetition and improve readability of your tests, as suggested in the checklist.